Glyssen/Bundle/GlyssenDblTextMetadata.cs

```diff
@@ -214,6 +218,7 @@ public void CopyGlyssenModifiableSettings(GlyssenDblTextMetadata source)
 			FontFamily = source.FontFamily;
```

Comment:
FontFamily and FontSizeInPoints, and ProjectStatus are also project settings. Should we maybe call this CharacterGroupGenerationPreferences instead?